sdk/servicebus/perf-tests/service-bus-track-1/test/receiveBatch.spec.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ReceiveMode, Receiver, Sender } from "@azure/service-bus";
+import { PerfOptionDictionary } from "@azure/test-utils-perf";
+import { ServiceBusTest } from "./sbBase.spec";
+
+interface ReceiverOptions {
+  "max-message-count": number;
+  "number-of-messages": number;
+  "message-body-size-in-bytes": number;
+}
+
+/**
+ * An underestimate of the true maximum size of a batch send message in bytes.
+ * This number is used to split up the send requests into small enough chunks
+ * to be accepted.
+ */
+const MAX_SEND_BATCH_SIZE_BYTES = 200000;
+
+export class BatchReceiveTest extends ServiceBusTest<ReceiverOptions> {
+  receiver: Receiver;
+  public options: PerfOptionDictionary<ReceiverOptions> = {
+    "number-of-messages": {
+      required: true,
+      description: "Total number of messages to send",
+      shortName: "max",
+      longName: "maximum",
```

Comment:
The names are same for both `"number-of-messages"` and `"max-message-count"`.
Can you update them?

Reply:
And add a command that uses all three options.
And also test it to make sure it's working please. 🙂

Reply:
This is also just copied from your track 2 test, I will update both.